dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -440,5 +448,117 @@ public void Test_Incremental_Job_Test_Pages_Are_Found_Create_And_Publish_New_Pag
 
     }
 
+    /**
+     * Given sceneario: Multi-language content referenced from a page. Create a Site-Search run including all the languages of the content and
+     * check the two versions made it into the resulting index.
+     */
+
+    @Test
+    public void test_MultilangContent_IndexingAllLanguages()
+            throws DotPublishingException, JobExecutionException, DotDataException, IOException, DotSecurityException, WebAssetException {
+
+        boolean defaultContentToDefaultLangOriginalValue =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'defaultContentToDefaultLangOriginalValue' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5260866)